src/pools/abstract-pool.ts

```diff
@@ -152,12 +143,12 @@ export abstract class AbstractPool<
    *
    * @param numberOfWorkers Number of workers that this pool should manage.
    * @param filePath Path to the worker-file.
-   * @param opts Options for the pool. Default: `{ maxTasks: 1000 }`
+   * @param opts Options for the pool. Default: `{}`
    */
   public constructor (
     public readonly numberOfWorkers: number,
     public readonly filePath: string,
-    public readonly opts: PoolOptions<Worker> = { maxTasks: 1000 }
+    public readonly opts: PoolOptions<Worker>
```

Comment:
```suggestion
    public readonly opts: PoolOptions<Worker> = {}
```

Reply:
This is not needed, all the implementations already have a default {}

Reply:
@Shinigami92  I would prefer to remove the default on the abstract pool since that the same default is already into any implementation ( fixed, dynamic, threads and cluster ).
We can remove this default from any implementation or from the abstract pool, does not make sense to have it duplicated between concrete and abstract class.
Removing this default from the abstract class, the concrete implementations can have different defaults if needed in future